scripts/jina-hub-update.py

```diff
@@ -0,0 +1,87 @@
+""" Script to change versioning of files (eg. manifest.yml) for executors
+[encoders, crafters, indexers, rankers, evaluators, classifiers etc.].
+It also adds the required jina version.
+Commits the change in the branch and raises a PR for the executor.
+"""
+import glob
+import os
+
+import git
+from github import Github
+from ruamel.yaml import YAML
+
+# TODO what env var do we export?
+g = Github(os.environ["GIT_TOKEN"])
+
+repo_dir = os.getcwd()
```

Comment:
not used?

Reply:
Oh yeah, good catch

Reply:
For some reason PyCharm doesn't detect it as unused